Tests/UnitTests/Purchasing/PackageTypeTests.swift

```diff
@@ -18,8 +18,16 @@ import XCTest
 
 @testable import RevenueCat
 
+@available(iOS 13.0, macOS 10.15, tvOS 13.0, watchOS 6.2, *)
 class PackageTypeTests: TestCase {
 
+    override func setUpWithError() throws {
+        try super.setUpWithError()
+
+        // iOS 12 does not support encoding top level objects as JSON
```

Comment:
Could you elaborate on this? 
how does the extension work in older versions?

Reply:
It works, we just can't test it with these as top level objects.
We'd have to test it encoding something like `[String: PackageType]`.

So solutions:
- Update test so it works in iOS 12 too (I didn't do it because I didn't think it was worth it)
- Improve comment to make that more clear

What do you think?

Reply:
@aboedo done.